client/signup/main.jsx

```diff
@@ -462,9 +464,9 @@ const Signup = React.createClass( {
 			<span>
 				<DocumentHead title={ this.pageTitle() } />
 				{
-					this.state.loadingScreenStartTime ?
-					null :
-					<FlowProgressIndicator
+					this.state.loadingScreenStartTime
+					? null
+					: <FlowProgressIndicator
```

Comment:
it would be a good time to replace this ternary with just a binary operation

```js
! this.state.loadingScreenStartTime && <FlowProgressIndicator …/>
```